src/constants.ts

```diff
@@ -84,7 +84,7 @@ export const STORAGE_KEYS = {
   LOCAL_STORAGE_LIBRARY: "excalidraw-library",
 };
 
-// time in milliseconds
+// Time in milliseconds
```

Comment:
let's not make a habit of this please — comments shouldn't be high school essays, and retroactive changes like these only clutter PRs and makes git blaming hard